pages/FileUpload.py

```diff
@@ -1,9 +1,8 @@
 import streamlit as st
```

Comment:
maybe we want to have separate FileUpload for FLASHViewer and FLASHTagger...

Reply:
I think it should be all moved to workflows so it can be run in the cloud (see `pages/FLASHDeconvWorkflow.py`). Then the `TOPP` tools create the output files that are used by the viewer.

Reply:
True. That would be a good feature. Especially for files >200MB if the users are not on windows (as thats the only executable available).

I agree with having multiple upload sections for each tool. I just separeted them out (already pushed) and will work on refactoring over the next couple of days.